src/kolibri_android/main_activity/kolibri_bus.py

```diff
@@ -49,6 +49,9 @@ def is_kolibri_url(self, url):
 
         return False
 
+    def can_transition(self, to_state: str) -> bool:
+        return (self.state, to_state) in self.transitions
```

Comment:
I see you are adding type hints here. These are probably the first ones in this repo. Fine with me!

Reply:
Oh oops, that was kind of by accident! Uh, doesn't hurt to leave them in I guess.